packages/builders/scripts/vats/terminate-governed-instance.js

```diff
@@ -0,0 +1,172 @@
+/**
+ * @file Source for a core-eval to terminate governed contract instances along
+ * with their governors by mapping a board ID to a key in the Bootstrap Powers
+ * governedContractKits collection.
+ * Also functions as an off-chain builder script for such core-evals:
+ *     agoric run /path/to/$0 <$instanceHandleBoardID:$instanceKitLabel>...
+ * TODO: Move out of builders/scripts/vats/, maybe to cosmic-swingset?
+ * https://github.com/Agoric/agoric-sdk/pull/10861#discussion_r1926208517
+ */
+
+/// <reference types="@agoric/vats/src/core/types-ambient"/>
+
+// dynamic import { makeHelpers } from '@agoric/deploy-script-support';
+// dynamic import { getSpecifier } from '@agoric/internal/src/module-utils.js';
+import { E } from '@endo/far';
+import { makeTracer } from '@agoric/internal';
+
+const USAGE = `Usage: agoric run /path/to/terminate-governed-instance.js \\
+  <$instanceHandleBoardID:$instanceKitLabel>...`;
+
+const repr = val =>
+  typeof val === 'string' || (typeof val === 'object' && val !== null)
+    ? JSON.stringify(val)
+    : String(val);
+const defaultMakeError = (strings, ...subs) =>
+  Error(
+    strings.map((s, i) => `${i === 0 ? '' : repr(subs[i - 1])}${s}`).join(''),
+  );
+const makeUsageError = (strings, ...subs) => {
+  const err = defaultMakeError(strings, ...subs);
+  console.error(err.message);
+  console.error(USAGE);
+  return err;
+};
+
+const rtarget = /^(?<boardID>board[0-9]+):(?<instanceKitLabel>.+)$/;
+/**
+ * @param {string[]} args
+ * @param {(strings: TemplateStringsArray | string[], ...subs: unknown[]) => Error} [makeError]
```

Comment:
is this here because `assert.Fail` isn't available in one of the environments where this code runs?
add a comment, perhaps?

Reply:
Yes, and that's a good idea.